README.md

```diff
@@ -5,182 +5,212 @@
+```
+docker compose exec ollama ollama pull all-minilm
+docker compose exec ollama ollama pull tinyllama
+```
 
-For other use cases, first [Install pgai](#installation) in Timescale Cloud, a pre-built Docker image, or from source. Then, choose your own adventure:
-  - Automate AI embedding with [pgai Vectorizer](/docs/vectorizer.md).
-  -  Use pgai to integrate AI from your provider. Some examples:
-     * [Ollama](./docs/ollama.md) - configure pgai for Ollama, then use the model to embed, chat complete and generate.
-     * [OpenAI](./docs/openai.md) - configure pgai for OpenAI, then use the model to tokenize, embed, chat complete and moderate. This page also includes advanced examples.
-     * [Anthropic](./docs/anthropic.md) - configure pgai for Anthropic, then use the model to generate content.
-     * [Cohere](./docs/cohere.md) - configure pgai for Cohere, then use the model to tokenize, embed, chat complete, classify, and rerank.
-     * [Voyage AI](./docs/voyageai.md) - configure pgai for Voyage AI, then use the model to embed.
-  - Leverage LLMs for data processing tasks such as classification, summarization, and data enrichment ([see the OpenAI example](/docs/openai.md)).
-  - Load datasets from Hugging Face into your database with [ai.load_dataset](/docs/load_dataset_from_huggingface.md).
+### Create a table and run a vectorizer
 
+1. **Connect to the database in your local developer environment**
+   The easiest way connect to the database is with the following command:
+   `docker compose exec -it db psql`. 
+   
+   Alternatively, you can connect to the database with the following connection string: `postgres://postgres:postgres@localhost:5432/postgres`.
 
+1. **Enable pgai on your database**
 
-## Installation
+    ```sql
+    CREATE EXTENSION IF NOT EXISTS ai CASCADE;
+    ```
+    
+1. **Create a table with the data you want to embed from a huggingface dataset**
 
-The fastest ways to run PostgreSQL with the pgai extension are to:
+    We'll create a table named `wiki` from a few rows of the english-language `wikimedia/wikipedia` dataset.
 
-1. Create your database environment. Either:
-   * [Use a pre-built Docker container](#use-a-pre-built-docker-container).
-   * [Use a Timescale Cloud service](#use-a-timescale-cloud-service).
-   * [Install from source](#install-from-source).
+    ```sql
+    SELECT ai.load_dataset('wikimedia/wikipedia', '20231101.en', table_name=>'wiki', batch_size=>5, max_batches=>1);
+    ```
+    
+    Related documentation: [load dataset from huggingface](/docs/load_dataset_from_huggingface.md).
+    
+    This table will contain the following columns: `id`, `url`, `title`, `text`. We'll create a primary key on the `id` column:
 
-2. [Enable the pgai extension](#enable-the-pgai-extension-in-your-database).
+    ```sql
+    ALTER TABLE wiki ADD PRIMARY KEY (id);
+    ```
```

Comment:
I'd love to A/B test this. My gut feeling is that creating a sample table with standard postgres DDL and clear manual inserts highlights better how native this integrates into postgres.

Including the huggingface download here is also another level of magic and therefore perceived complexity. It's great for e.g. a video demo or blog post when someone is already involved but for trying to understand this project at a glance... I'm not convinced this is a good idea.

(again inserts could be hidden behind a `<details>` tag)